### ISSUE TITLE
[Feature Request] Passively save my location history whenever possible (e.g. the phone is awaken), then send all of it on request

### ISSUE THREAD

**Issue by tdbe** [Seva-coder/Finder#38]
Not sure if without google services you can figure this out on android. But I think it would be amazing if this app had an option to track and store your route locally - but passively (e.g. to not actively awaken phone every minute to connect to gps and save a point, but rather to wait for either 1). the next time the phone is manually awaken by user, or 2). take advantage of any other automatic awake timers or events that android receives from other apps and other services, to also have Finder check for location (e.g. you get a new instant message and the screen lights up? Cool. Finder will also record a location point).

Basically the motivation is to 1). have a personal log of where I've been, but without (that much) extra battery drain. And 2). whenever someone requests my location via SMS, my phone can send them all my locations for the past e.g. 8 hours instead of just one point. 

Would also gladly pay for such a feature. It would be super useful, especially when you're taking care of an older person.

Cheers.

**Comment by Seva-coder** [comment]
as I understand, most of api for such function is in google play services (which is not open...). But it analog can be created in service, which will run in background and log locations/networks (when they available). Internet accessibility makes this task easier - app can use wifi to get locations, and in sms send just lat/lon/timestamp (it take less space in sms)
Anyway, feature is not simple to code it. But such function can be usefull, may be it will be added in one of next versions...
